packages/experimental/src/components/ContextHeader/_index.scss

```diff
@@ -1,7 +1,8 @@
 //
-// Copyright IBM Corp. 2021, 2021
+// Copyright IBM Corp. 2020, 2020
```

Comment:
This looks a bit odd, but I think it's just because I copied/renamed the file from elsewhere. All the other files in this component are marked 2020, 2020.